packages/astro/src/cli.ts

```diff
@@ -1,12 +1,11 @@
 /* eslint-disable no-console */
-import type { AstroConfig } from './@types/astro';
-
-import * as colors from 'kleur/colors';
 import { promises as fsPromises } from 'fs';
+import * as colors from 'kleur/colors';
 import yargs from 'yargs-parser';
-
-import { loadConfig } from './config.js';
+import { z } from 'zod';
```

Comment:
I hadn't seen the `zod` library before, this is very cool.  Their docs start with "zero dependencies" and "functional approach", can't beat that!